packages/zwave-js/src/lib/driver/NetworkCache.ts

```diff
@@ -97,6 +97,7 @@ export const cacheKeys = {
 			defaultVolume: `${nodeBaseKey}defaultVolume`,
 			lastSeen: `${nodeBaseKey}lastSeen`,
 			deviceConfigHash: `${nodeBaseKey}deviceConfigHash`,
+			rate: `${nodeBaseKey}rate`,
```

Comment:
I also wouldn't cache that information. It will self-populate again whenever there is communication.

`lastSeen` is cached because we use it to decide in which order nodes are pinged after startup.

Reply:
Done, removed